getblocklab/block-lab
# Bump server-side-render npm version, attempting to fix an error

#### Changes
* Bumps the version of [@wordpress/server-side-render](https://www.npmjs.com/package/@wordpress/server-side-render) to the latest version

#### Background
2 people have [reported](https://wordpress.org/support/topic/json-error-in-ui-when-using-certain-words/#post-12524614) a similar error message to https://github.com/WordPress/gutenberg/issues/18419:

```
Uncaught TypeError: Cannot set property ServerSideRender of #<Object> which has only a getter
```

This might not fix it, but it's probably a good first effort

#### Testing instructions
(Regression testing for the `<ServerSideRender>` component)
1. Create a new post, and add a BL block with several fields
2. Ensure the `<ServerSideRender>` displays as expected
